packages/arb-validator-core/ethbridgetest/message_test.go

```diff
@@ -30,230 +40,350 @@ import (
 	"github.com/offchainlabs/arbitrum/packages/arb-validator-core/test"
 )
 
-func init() {
+var privateKeyHex = "27e926925fb5903ee038c894d9880f74d3dd6518e23ab5e5651de93327c7dffa"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 86 characters, out of limit 80](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)